src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -712,8 +711,8 @@ struct overwrite_plan_t {
       left_operation(overwrite_operation_t::UNKNOWN),
       right_operation(overwrite_operation_t::UNKNOWN),
       block_size(block_size),
-      is_left_stable(pins.front()->is_stable()),
-      is_right_stable(pins.back()->is_stable()) {
+      is_left_fresh(pins.front()->is_stable()),
+      is_right_fresh(pins.back()->is_stable()) {
```

Comment:
It shouldn't be renamed directly, e.g: `is_left_fresh` should be assigned as `!pins.front()->is_stable()` here, and the below logic should check `} else if (is_left_fresh) {` instead.

Reply:
Done.